main.go

```diff
@@ -33,6 +33,8 @@ func run() error {
 		Addr:    cfg.APIHost,
 		Handler: routers.NewAPI(cfg, db),
 	}
+
+	fmt.Print("Running the server...")
```

Comment:
Prob should remove

Reply:
It has been helpful for me in bugfixing w / dockerfiles and similar. why would it require removal

Reply:
Fair